news/2025/2025-05-04-featured-artist-track-updates-krimek.md

```diff
@@ -0,0 +1,73 @@
+        <source src="https://assets.ppy.sh/media/news/krimek_updates.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [Krimek feat. eili & Rafterz - Michi no Zankyou](https://assets.ppy.sh/artists/371/Golden%20Tears%20EP/Krimek%20-%20Michi%20no%20Zankyou%20(feat.%20eili%20%26%20Rafterz).osz)
+
+Play through [**Krimek**'s submission to Kyoku 2024](https://osu.ppy.sh/beatmapsets/2242273) mapped by the artist!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/371/Golden%20Tears%20EP/Krimek%20-%20Michi%20no%20Zankyou%20(feat.%20eili%20%26%20Rafterz).mp3">
+</audio>
+
+### [Krimek feat. michiri9 - Lunaria](https://assets.ppy.sh/artists/371/Lunaria/Krimek%20feat.%20michiri9%20-%20Lunaria.osz)
+
+Try out [this map from last year's Resurrection Cup](https://osu.ppy.sh/beatmapsets/2215066) hosted by [Mimiliaa](https://osu.ppy.sh/users/7117621)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/371/Lunaria/Krimek%20feat.%20michiri9%20-%20Lunaria.mp3">
+</audio>
+
+### [Krimek - Coeval Era](https://assets.ppy.sh/artists/371/Coeval%20Era/Krimek%20-%20Coeval%20Era.osz)
+
+Experience [DACH Open 2024's Grand Finals tiebreaker](https://osu.ppy.sh/beatmapsets/2214965) mapped by [Pho](https://osu.ppy.sh/users/3624692), [Lince Cosmico](https://osu.ppy.sh/users/6070370), [Megafan](https://osu.ppy.sh/users/6632605), and [MaestroSplinter](https://osu.ppy.sh/users/6707918)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/371/Coeval%20Era/Krimek%20-%20Coeval%20Era.mp3">
+</audio>
+
+### [Krimek feat. eili - Silent Sentinel](https://assets.ppy.sh/artists/371/Silent%20Sentinel/Krimek%20feat.%20eili%20-%20Silent%20Sentinel.osz)
+
+Here's the [Vietnam osu! Championship 2024's Knockout 2 tiebreaker](https://osu.ppy.sh/beatmapsets/2254625) mapped by [Typ4](https://osu.ppy.sh/users/6902361), [Ducky-](https://osu.ppy.sh/users/9351565), [nik](https://osu.ppy.sh/users/10077264), [Nagabi](https://osu.ppy.sh/users/10352099), [Vermasium](https://osu.ppy.sh/users/11106442), and [Arushii](https://osu.ppy.sh/users/15664628)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/371/Silent%20Sentinel/Krimek%20feat.%20eili%20-%20Silent%20Sentinel.mp3">
+</audio>
+
+### [Krimek - Revival Of A New Master](https://assets.ppy.sh/artists/371/Revival%20Of%20A%20New%20Master/Krimek%20-%20Revival%20Of%20A%20New%20Master.osz)
+
+And here's [a map from Resurrection Cup 2023](https://osu.ppy.sh/beatmapsets/2028363) also mapped by the artist!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/371/Revival%20Of%20A%20New%20Master/Krimek%20-%20Revival%20Of%20A%20New%20Master.mp3">
+</audio>
+
+---
+
+If you're waiting for a sign before you start mapping, this is it. More specifically, every **NEW** tag beside songs on [**Krimek**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/371) is your sign. Pick one out and circle placing instead of circle clicking for a change.
```

Comment:
```suggestion
If you're waiting for a sign before you start mapping, this is it. More specifically, every **NEW** tag beside songs on [**Krimek**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/371) is your sign. Pick one out and start circle-placing instead of circle-clicking for a change.
```